src/sql/workbench/browser/modelComponents/loadingComponent.component.ts

```diff
@@ -12,20 +12,23 @@ import * as azdata from 'azdata';
 
 import { ComponentBase } from 'sql/workbench/browser/modelComponents/componentBase';
 import { IComponent, IComponentDescriptor, IModelStore } from 'sql/workbench/browser/modelComponents/interfaces';
-import * as nls from 'vs/nls';
+import { localize } from 'vs/nls';
 
 @Component({
 	selector: 'modelview-loadingComponent',
 	template: `
-		<div class="modelview-loadingComponent-container" role="alert" aria-busy="true" *ngIf="loading">
-			<div class="modelview-loadingComponent-spinner" *ngIf="loading" [title]=_loadingTitle #spinnerElement></div>
+		<div role="status" aria-live="polite" class="modelview-loading-component-status-message">
```

Comment:
No alert role?  I found that setting aria-live wasn't enough unless the contents of the div themselves were changing after the components were loaded.

Reply:
I am using status role now, it is also a live region role similar to alert. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_status_role